### ISSUE TITLE
Resizing windows other than in the main window is hard

### ISSUE THREAD

**Issue by dschemp** [PolyMC/PolyMC#379]
### Operating System

Linux

### Description of bug

It is really hard to find the edge on the window when creating a new instance or in the settings to resize the window. At one point I thought the windows were unresizable. This was an issue to me as I think the "create an instance" window is far to small by default regarding the list of MC versions and modpacks displayed.

The main window offers a little triangle in the bottom right hand corner which helps quite a lot.

I used PolyMC Version 1.1.1 provided through Flathub on Fedora Linux 35 with GNOME.

### Steps to reproduce

Open any subwindow to the main window (like create an instance).

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by 0wM** [comment]
same
https://www.youtube.com/watch?v=t54I1iE6jds

**Comment by jn64** [comment]
Is this still an issue on latest PolyMC? In my experience (also F35/GNOME) the resize handles are identical between the main window and any dialogs.

Anyway I have an external solution for you. GNOME has a _window action key_ (default: Super) which you can use to easily move/resize windows.

- Hold Super + leftclick anywhere within a window to move it
- Hold Super + middleclick within a window to resize from the nearest edge/corner

In this way you never have to hunt for the resize handle on window borders again.

You can change the action key or change resize to rightclick using gnome-tweaks (GUI) or gsettings (in the schema `org.gnome.desktop.wm.preferences`, relevant keys are `mouse-button-modifier` and `resize-with-right-button`).

**Comment by dschemp** [comment]
> Anyway I have an external solution for you. GNOME has a window action key (default: Super) which you can use to easily move/resize windows.

Yeah, I know about that and regularly use this feature. It's awesome but shouldn't be the "solution" to this.
And unfortunately this problem persists for me (now on Fedora Linux 36 and PolyMC 1.3.1). It's hard to get grasp of the edges and corners and any windows other than the main window has no "resize triangle" in the bottom right.

**Comment by jn64** [comment]
Ok, I can't reproduce the resize handles not appearing.

I also can't reproduce the new instance window being too small in the first place. Temporarily deleting `polymc.cfg` (as if it's a new install), the new instance window is a reasonable size (as are all other windows I could find).

Have you tried X11 vs Wayland?

**Comment by jn64** [comment]
Maybe this [rhbz#1919584 Invisible resize border is broken for many (all?) Qt/KDE apps, making their windows unnecessarily painful to resize under GNOME 3](https://bugzilla.redhat.com/show_bug.cgi?id=1919584)

related [rhbz#1759259](https://bugzilla.redhat.com/show_bug.cgi?id=1759259), [QTBUG-68636](https://bugreports.qt.io/browse/QTBUG-68636)

qgnomeplatorm qt5-qtwayland

**Comment by jn64** [comment]
The above bug(s) should be fixed in [qgnomeplatform-0.8.4-12.fc36 and qt5-qtwayland-5.15.5-4.fc36 available today](https://bodhi.fedoraproject.org/updates/FEDORA-2022-69bf07b7ea).
